src/annotator/guest.js

```diff
@@ -135,11 +139,11 @@ export default class Guest {
     this._adder = new Adder(this.element, {
       onAnnotate: async () => {
         await this.createAnnotation();
-        /** @type {Selection} */ (document.getSelection()).removeAllRanges();
+        removeTextSelection();
```

Comment:
The `createAnnotation` method makes its own call to `removeTextSelection`. Can you help me understand why these additional calls are necessary after the returned Promise resolves?

Reply:
Good point, they are not needed.

There are two different paths for calling `Guest#createAnnotation`, through the adder and through an RPC call from the host frame (by clicking `New annotation` or `New page note` button).

I will remove the calls from the adder and remove the selected text, if any, only in `Guest#createAnnotation`.